app-scientific/sagemath/spec

```diff
@@ -0,0 +1,6 @@
+VER=10.4
+COMMIT_ID_LOONGARCH64="f010913d53dc2758a360143f0b75c9b5b5581ecc"
+SRCS="git::commit=tags/$VER::https://github.com/sagemath/sage"
+CHKSUMS="SKIP"
+SRCS__LOONGARCH64="git::commit=$COMMIT_ID_LOONGARCH64::https://github.com/lwzhenglittle/sage-loongarch.git"
+CHKSUMS__LOONGARCH64="SKIP"
```

Comment:
Missing CHKUPDATE:

https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md

We prefer using `anitya` where possible.